imgui/core.pyx

```diff
@@ -1374,6 +1374,68 @@ def is_window_collapsed():
     return cimgui.IsWindowCollapsed()
 
 
+def set_tooltip(str text):
+    """Set tooltip under mouse-cursor.
+    Usually used with :func:`is_item_hovered()`.
+    For a complex tooltip window see :func:`begin_tooltip()`.
+
+    .. visual-example::
+        :auto_layout:
+        :height: 100
+        :width: 200
+        :click: 10 10
```

Comment:
If you set click position to something like `80 40` we will be able to actually show the tooltip in the documentation example.